src/handlers.js

```diff
@@ -219,13 +217,14 @@ async function statusHandler(message) {
   }
 
   debug(`Attempting to update status for ${build.organization}/${build.repository}@${build.sha} (${state})`);
+  const target_url = `${this.context.cfg.taskcluster.rootUrl}/task-group-inspector/#/${taskGroupId}`;
```

Comment:
You can use the `ui` function of `taskcluster-lib-urls` to generate this (it does about the same thing, but more cleanly).

Reply:
This seems to be simpler to change and debug tho... If there ever be something wrong with this url - you can immediately see where it's coming from